src/WebJobs.Script.WebHost/Handlers/WebScriptHostHandler.cs

```diff
@@ -47,16 +42,6 @@ public WebScriptHostHandler(HttpConfiguration config, int hostTimeoutSeconds = H
                 request.SetProperty(ScriptConstants.AzureFunctionsHttpRequestAuthorizationDisabledKey, true);
             }
 
-            // some routes do not require the host to be running (most do)
-            bool bypassHostCheck = request.MatchRoute("admin/host/status");
```

Comment:
This was the single exclusion. Lots of other APIs (key management, etc.) should have also fallen into this bucket.